formulaire-public/src/main/resources/public/ts/directives/public-question-item.ts

```diff
@@ -49,6 +49,14 @@ class Controller implements IViewModel {
                 this.mapChoiceResponseIndex.set(choice, this.responses.all.indexOf(matchingResponses[0]));
             }
         }
+
+        if (this.question.question_type === Types.CURSOR) {
+            this.responses.all[0].answer =
+                this.responses.all[0].answer == null ?
+                    this.question.cursor_min_val : // If answer doesn't exist we initialize it
+                    Number.parseInt(this.responses.all[0].answer.toString()); // If answer exist we parse it to a NUMBER
```

Comment:
Une proposition si tu veux géré le Nan, au cas ou par miracle this.responses.all[0].answer ne sois pas un nombre
```suggestion
            let answer = Number.parseInt(this.responses.all[0].answer)
            this.responses.all[0].answer = (Number.isInteger(answer) ? answer : this.question.cursor_min_val)
```